cfme/infrastructure/resource_pool.py

```diff
@@ -5,24 +5,108 @@
            Resource pool pages.
 """
 from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic.widget import View
+from widgetastic.exceptions import NoSuchElementException
+from widgetastic_patternfly import Button, Dropdown, FlashMessages
 
-from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import Quadicon, Region, toolbar as tb, paginator, match_location
-from functools import partial
+from cfme.base.ui import BaseLoggedInPage
+from cfme.exceptions import ResourcePoolNotFound
+from cfme.web_ui import Quadicon, match_location
```

Comment:
We could probably remove Quadicon since I see no tests that are actually using the function that is using Quadicon, but I used match_location. Is there a Widgetastic equivalent?

``python
class ResourcePoolView(BaseLoggedInPage):
    """Base view for header and nav checking, navigatable views should inherit this"""
    @property
    def in_resource_pool(self):
        nav_chain = ['Compute', 'Infrastructure', 'Resource Pools']
        return (
            self.logged_in_as_current_user and
            self.navigation.currently_selected == nav_chain and
            # TODO: Also needs to be converted to Widgetastic
            match_location(controller='resource_pool', title='Resource Pools'))
```